### ISSUE TITLE
Add tests for repo/vcs.go

### ISSUE THREAD

**Issue by aaronkavlie-wf** [Masterminds/glide#341]
`repo/vcs.go` is lacking tests entirely. This makes changes scary.

**Comment by mattfarina** [comment]
@aaronkavlie-wf I agree. Most of the underlying work (which included tests) ended up in [`github.com/Masterminds/vcs`](https://github.com/Masterminds/vcs) which has tests. That said, we need more tests here. It would save me on all the manual tests I run when changes are made here.

Pull requests are welcome.

**Comment by aaronkavlie-wf** [comment]
Could those manual tests be turned into an integration test script that others could run? That would go a long way.
